en/02_Developer_Guides/14_Files/07_File_Usage.md

```diff
@@ -59,7 +59,7 @@ class UsedOnTableExtension extends Extension
     // DataObject the File is used on.  This is useful for two reasons:
     // - It gives context more context, for instance if File is used on a Content block, it can be used to show the
     //   Page that Content Block is used on
-    // - The DataObject may not have a `CMSEditLink()` implementation, though the ancestor DataObject does.
+    // - The DataObject may not return a value from the `CMSEditLink()` method, though the ancestor DataObject does.
```

Comment:
Why not? Now that the method is directly on DataObject you can just implement it on your subclass?

Reply:
The default implementation doesn't return a value. Yes you _can_ implement it on your subclass. But you don't have to.